src/components/home/movie-row.tsx

```diff
@@ -25,37 +32,74 @@ const MovieRow: React.FC<MovieRowProps> = ({
     slider.scrollLeft = slider.scrollLeft + 500;
   };
 
+  const [showSearch, setShowSearch] = useState(false);
+
   return (
     <>
-      <h2
-        className="text-white my-5 sm:mx-9 sm:text-lg md:text-xl lg:text-2xl font-bold px-2 antialised tracking-wider"
-        id="nav-title"
-      >
-        {yourUsername === username
-          ? "Your Picks"
-          : data.moviesByUser[username].displayName}
-      </h2>
-      <div className="relative flex items-center group">
-        <div className="group relative hidden sm:block h-full">
+      {showSearch && (
+        <SearchPalette
+          username={username}
+          showSearch={showSearch}
+          setShowSearch={setShowSearch}
+          mutateUserData={mutateUserData}
+        />
+      )}
+
+      <div className="grid grid-cols-12 my-6 h-full">
+        <div></div>
+        <div></div>
```

Comment:
why do u have two empty tags?

Reply:
u should clean your code before making a PR bud